deegree-tools/deegree-tools-gml/src/main/java/org/deegree/tools/featurestoresql/loader/GmlLoaderHelpUsage.java

```diff
@@ -34,11 +35,14 @@ public static void printUsage() {
         System.out.println();
         System.out.println( "arguments:" );
         System.out.println( " -pathToFile=<path/to/gmlfile>, the path to the GML file to import" );
+        System.out.println( " -pathToList=<path/to/listfile>, the path to the file containing the files to import (one path per line. lines starting with # will be ignored)" );
         System.out.println( " -workspaceName=<workspace_identifier>, the name of the deegree workspace used for the import. Must be located at default DEEGREE_WORKSPACE_ROOT directory" );
         System.out.println( " -sqlFeatureStoreId=<feature_store_identifier>, the ID of the SQLFeatureStore in the given workspace" );
         System.out.println();
         System.out.println( "options:" );
         System.out.println( " -disabledResources=<urlpatterns>, a comma separated list url patterns which should not be resolved, not set by default" );
+        System.out.println( " -chunkSize=<features_per_chunk>, number of features processed per chunk");
+        System.out.println( " -dryRun=true, skip writeing features (only check if reading is possible)");
```

Comment:
Typo in "writeing"

Reply:
Wording has been changed to:
` -dryRun=true, enable dry run where writing is skipped (checks only if all data can be read), disabled by default`